conf.xml.tmpl

```diff
@@ -175,13 +175,11 @@
                     Trigger for registering eXists XML:DB URL handler with Java
                 -->
                 <trigger class="org.exist.protocolhandler.URLStreamHandlerStartupTrigger">
-                    <!-- Keep stream's data in memory -->
+                    <!-- Keeps stream data on disk (temporary files are used for XML documents) -->
```

Comment:
one of the elements is not commented out. Is this OK?

Reply:
Yup.